examples/exotica_examples/scripts/example_ik_eff_axis_alignment.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import rospy
+import pyexotica as exo
+import numpy as np
+from numpy import array
+from numpy import matrix
+import math
+from pyexotica.publish_trajectory import *
+from time import sleep
+import task_map_py
+from sensor_msgs.msg import Joy, JointState
+from std_msgs.msg import Float64MultiArray as FloatArray
+
+rospy.init_node('example_ik_eff_axis_alignment_node')
+
+DT = 1.0/100.0 # 100 HZ
+DAMP = 0.005
+
+XLIM = [0.0, 0.75]
+YLIM = [-0.5, 0.5]
+ZLIM = [0, 1.0]
+
+LIMITS = array([XLIM,\
+                YLIM,\
+                ZLIM])
+
+class Example(object):
+
+    def __init__(self):
+
+        exo.Setup.initRos()
+        (sol, prob)=exo.Initializers.loadXMLFull('{exotica_examples}/resources/configs/example_eff_axis_alignment.xml')
+        self.problem = exo.Setup.createProblem(prob)
+        self.solver = exo.Setup.createSolver(sol)
+        self.solver.specifyProblem(self.problem)
+        self.scene = self.problem.getScene()
+        
+        self.q=array([0.0]*7)
```

Comment:
Python formatting - e.g. PEP8 without the linebreaks recommended (inconsistent).